v63003/outref/gtm8735.txt

```diff
@@ -33,40 +33,40 @@ Database file ##TEST_PATH##/mumps.dat now has BG access method
 # Displaying status of ##SOURCE_PATH##/dsehelp.gld
   LOCK shares DB critical section     FALSE  Read Only                       ON
 # Attempting to change to no read only
-%YDB-E-DBFILOPERR, Error doing database I/O to database file ##SOURCE_REALPATH##/dsehelp.dat
+%YDB-E-DBFILOPERR, Error doing database I/O to database file ##SOURCE_PATH##/dsehelp.dat
```

Comment:
This was SOURCE_REALPATH before but is now SOURCE_PATH. It has to be the former or else this test will fail on systems where the test output directory is a soft link.